app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,38 @@
  *= require font-awesome
  */
 
+#map-container { 
+  border-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.